tutorials/migration.md

```diff
@@ -292,134 +237,112 @@ To get a list of all currently installed items run:
+Make note of those that do not appear in the list. 
+Do not install any binding that has a version 2.x at this time. 
+If there is not a 1.x.0 version we will install the binding manually (see below).
 
-When you find a binding to install, press the Install button and wait for
-installation to complete. Do not install the My.openHAB Binding yet.
+This is a great opportunity to identify those bindings you actually use and only install those.
 
-If you are running openHAB in a Docker container you need to create
-`<openHAB home>/userdata/persistence` manually prior to installing a persistence
-add-on.
+When you find a binding to install, press the Install button and wait for installation to complete. 
+Do not install the myopenhab/openhabcloud Binding yet.
 
-One thing to be aware of is any configuration done in PaperUI or Habmin gets
-saved to a non-human readable database. And since none of the administration
-UIs are currently capable of doing everything, you will end up with a mix of
-database and text based configuration. This is why we recommend not using
-PaperUI for those migrating to openHAB 2 from openHAB 1.x.
+One thing to be aware of is any configuration done in PaperUI gets saved to a files and folders under <openHAB 2 conf>.
+Binding configurations, Items, Things, et. al. created in, edited in, and otherwise modified in PaperUI will not appear in the conf directories.
+For example, if you create an Item in PaperUI, you will not find that new Item in any of the .items files.
+Instead they get saved to a JSONDB file located in <openHAB 2 userdata>/jsondb.
+Since none of the administration UIs are currently capable of doing everything and we want to minimize the number of changes necessary, you will ultimately end up with a mix of database and text based configuration. 
+This is why we recommend not using PaperUI for those migrating to openHAB 2 from openHAB 1.x.
 
 ### Configure Add-ons
 
-As bindings and other add-ons are installed, you can watch for errors in the
-logs. Errors at this time may not be important but make note of those that did
-generate errors during installation for special attention later.
+As bindings and other add-ons are installed, you can watch for errors in the logs. 
+Errors at this time may not be important but make note of those that did generate errors during installation for special attention later.
 
-Once a binding or add-on is installed, it will create a
-`<openHAB 2 conf>/services/<add-on name>.cfg` file. Unlike in openHAB 1.x where all the
-binding configurations are placed in the one openhab.cfg file, openHAB 2 has a
-separate .cfg file for each binding. Transfer the settings for each binding from
-openhab.cfg to its new .cfg file, removing the binding name from the parameter.
-For example, the parameter `nest:refresh=300000` in openhab.cfg becomes
-`refresh=300000` in nest.cfg.
+Once a binding or add-on is installed, it will create a `<openHAB 2 conf>/services/<add-on name>.cfg` file. 
```

Comment:
According to #189 we should use a consistent `$FOLDER` notation for all articles.